### DISCUSSION TITLE
RBA use cases

### DISCUSSION THREAD

**Discussion by ZachTheSplunker** [splunk/rba#20] [Q&A]
>Question from "The Office" hours meetup 

What are some useful use cases?

**Comment by ZachTheSplunker** [comment]
>Answer from "The Office" hours meetup

## These are some uncommon but useful ones
- Multiple fraud behaviors on a single fraud object [ss# that didn't exist, unmatched phone #, etc]
- UBA for employee retirement system (critical business application)

**Comment by SweetTNC** [comment]
Great example of Fraud use case:

- https://www.linkedin.com/pulse/pandemic-unemployment-assistance-fraud-chris-perkins/
- https://www.linkedin.com/pulse/pandemic-unemployment-assistance-fraud-zero-trust-part-chris-perkins/
- https://www.linkedin.com/pulse/pandemic-unemployment-assistance-fraud-zero-trust-part-chris-perkins-1c/

Splunk Fraud Analytics App is RBA based app - https://splunkbase.splunk.com/app/6250#/overview
